components/brave_wallet_ui/page/screens/composer_ui/to_asset/to_asset.tsx

```diff
@@ -111,7 +114,10 @@ export const ToAsset = (props: Props) => {
 
   // render
   return (
-    <ToSectionWrapper fullWidth={true}>
+    <ToSectionWrapper
+      tokenColor={getDominantColorFromImageURL(token?.logo ?? '')}
```

Comment:
perf: This should probably be memoized since there is a decent amount of computation in this function.

Reply:
Is now memoized!